linkerd/app/inbound/src/policy/http.rs

```diff
@@ -81,16 +81,30 @@ where
         HttpPolicyService {
             target,
             policy,
-            meta: ConnectionMeta { client, dst, tls },
+            connection: ConnectionMeta { client, dst, tls },
             metrics: self.metrics.clone(),
             inner: self.inner.clone(),
-            default_route_meta: self.default_route_meta.clone(),
         }
     }
 }
 
 // === impl HttpPolicyService ===
 
+macro_rules! err {
+    ($e:expr) => {
+        return future::Either::Right(future::err($e))
+    };
+}
+
+macro_rules! try_fut {
+    ($e:expr) => {
+        match $e {
+            Ok(v) => v,
+            Err(e) => err!(e),
+        }
+    };
+}
```

Comment:
occurs to me that there's probably some code elsewhere in the proxy that could be cleaned up with these...